src/builder.rs

```diff
@@ -97,6 +98,10 @@ pub struct Builder<'fbb> {
     temp_traversal: Vec<TempTraversal>,
     // Temporary [`Anchor`]s because we need to project them on the [`Traversal`] of each LRM they belong to.
     temp_anchors: Vec<AnchorPosition>,
+    // Position of every node
+    nodes_coords: Vec<Coord>,
+    // Every node of a given traversal
+    nodes_of_traversal: Vec<Vec<usize>>,
```

Comment:
I don't understand (but I'm completely new to the project): this field seems to only be read, never pushed to. So what is the purposed if it's always empty? What do I miss?

Reply:
Sorry, I messed up with my chained pull requestes. This is a rest of an other, and with the bases that have been changed.

The nodes are added in https://github.com/osrd-project/liblrs/pull/53/files#diff-e4f794ca308aa712cbad31feb714df7cada1bbe453a30e232e45891571430263R231

But this is not relevant to this PR and it should be clearer now